client/ctclient/ctclient.go

```diff
@@ -216,6 +219,63 @@ func getInclusionProofForHash(ctx context.Context, logClient *client.LogClient,
 	}
 }
 
+func getConsistencyProof(ctx context.Context, logClient *client.LogClient) {
+	if *treeSize <= 0 {
+		log.Fatal("No valid --size supplied")
+	}
+	if *prevTreeSize <= 0 {
+		log.Fatal("No valid --prev_size supplied")
+	}
+	var hash1, hash2 []byte
+	if *prevHash != "" {
+		var err error
+		hash1, err = hex.DecodeString(*prevHash)
+		if len(hash1) != 32 {
```

Comment:
Aren't you going to lose the useful error returned by decodestring here which might give a clue to the users to why the hash is 0 bytes rather than 32?

Same below